### DISCUSSION TITLE
Proposal to Move Swanchain Governance Discussions to a Social Forum

### DISCUSSION THREAD

**Discussion by abbakaka** [swanchain/governance#2] [General]
I would like to propose a shift in how we handle governance discussions within the swanchain ecosystem, even thought is a new mechanisms and is just a beginning governance processes and discussions take place on GitHub, surely gitHub is a powerful tool, it can be challenging for many community members who are not technically inclined or familiar with its interface

If we want a  broader community participation in the ecosystem discussions and make it easier for all community members to contribute to governance, I suggest that we consider moving these discussions to a more user-friendly platform, possible options could include setting up a dedicated forum like gov.swanchain.org or forum.swanchain.org.

I would love to hear your thoughts and feedback on this suggestion. If there is broad support, we can explore the logistical aspects of making this shift, i have been a DAO council member in NEAR protocol for more than 2yrs specifically Ecosystem Marketing and growth DAO and core contributor of NDC as well as African DAO co-founder.

**Comment by flyworker** [comment]
Thanks for your advice,do you have some candidate as the software? We choose Github because most devs already have a github account and easy to manage.

**Reply by abbakaka** [reply]
Yeah - -  GitHub is widely used by developers and very convenience for managing contributions. but if we want a  broader participation and make governance more accessible to all swanchain community members like i said, i suggest utilizing the user friendly platforms for easy navigations, such as gov.swanchain.org,  forum.swanchain.org or community.swanchain.io

Examples; 
- https://gov.near.org/latest
- https://community.iotex.io/

**Reply by flyworker** [reply]
It seems using Discourse, I can ask our devs to see the timeline and how to fit it into.
Thanks for the advice!

**Reply by tinaks01** [reply]
lfg

**Comment by chibyke71** [comment]
Agreed. I find GitHub can be somewhat cumbersome and confusing to navigate, which might hinder regular community members who aren't developers from engaging. To encourage broader participation, I believe a website platform for governance discussions would be ideal, as most people are familiar with navigating such platforms, and other communities successfully use them.

**Comment by gugege521** [comment]
我在想究竟哪个平台可以真实且公平反应社区成员的意见呢？也就是说未来社区成员参与治理的权重是否直接源于持币的数量呢？如果是的话，这个平台必须能够和钱包挂钩哈，最好还可以和交易所的地址挂钩。

**Comment by flyworker** [comment]
I think it is good idea to build a new forum, but I think it fit our current need at the moment. Manage the forum is some extra work for the team at the moment, when there are more discussions ,we can do it
